deeds-dapp-webapp/src/main/webapp/vue-app/staticPage/components/StaticPageContent.vue

```diff
@@ -18,6 +18,10 @@
 -->
 <script>
 export default {
-  template: document.getElementById('staticContent').innerHTML.replace(/images\//g, `/${window.location.pathname.split('/')[1]}/static/images/`),
+  template: document.getElementById('staticContent').innerHTML,
```

Comment:
why changing the strategy of images ?

Reply:
I need the parentLocation to redirect the user to /owners and /portfolio

Reply:
The fact that you need `parentLocation` doesn't mean we need to make adding images in html content, something complex.